Neos.ContentGraph.DoctrineDbalAdapter/Classes/Domain/Repository/ContentGraph.php

```diff
@@ -183,8 +206,8 @@ public function findNodeAggregateByIdentifier(
             return null;
         }
 
-        $rawNodeTypeName = null;
-        $rawNodeName = null;
+        $rawNodeTypeName = '';
+        $rawNodeName = '';
```

Comment:
PHPStorm mainly, which (correctly imho) states that those might not be changed to string at all. Not that it matters much since the VO doesn't allow empty string either, but it's a bit cleaner this way.

Reply:
thanks for clarifying :)